src/sparse_blas/backends/rocsparse/rocsparse_task.hpp

```diff
@@ -0,0 +1,44 @@
+/***************************************************************************
+*  Copyright (C) Codeplay Software Limited
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  For your convenience, a copy of the License has been included in this
+*  repository.
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+**************************************************************************/
+
+#ifndef _ONEMKL_SPARSE_BLAS_BACKENDS_ROCSPARSE_TASK_HPP_
+#define _ONEMKL_SPARSE_BLAS_BACKENDS_ROCSPARSE_TASK_HPP_
+
+#include "rocsparse_error.hpp"
+#include "sparse_blas/backends/common_launch_task.hpp"
+
+namespace oneapi::mkl::sparse::rocsparse::detail {
```

Comment:
Other than cusparse -> rocsparse, this file, rocsparse/rocsparse_task.hpp appears to be nearly identical to cusparse/cusparse_task.hpp. Is there anything more than that?

Reply:
Yes, they are almost identical. I'm not sure I understand, why do you ask?

Reply:
I was just confirming for my understanding to make sure I wasn't missing anything.